contracts/StablecoinConverter.sol

```diff
@@ -297,6 +298,7 @@ contract StablecoinConverter is EpochTokenLocker {
         grantRewardToSolutionSubmitter(burntFees);
         tokenConservation.checkTokenConservation();
         documentTrades(batchIndex, owners, orderIds, volumes, tokenIdsForPrice);
+        return (latestSolution.objectiveValue);
```

Comment:
it is always cheaper to read from memory variables than from storage variables. 
Maybe you store `utility - disregardedUtility + burntFees` in memory and then return it?

Reply:
Makes sense, I didn't think about this.